graphql_compiler/tests/test_compiler.py

```diff
@@ -4341,6 +4340,38 @@ def test_fold_on_output_variable(self):
               Animal___1.name AS `animal_name`,
               [x IN collected_Animal__out_Animal_ParentOf___1 | x.name] AS `child_names_list`
         '''
+        expected_mssql = '''
+            SELECT
+              [Animal_1].name AS animal_name,
+              folded_subquery_1.fold_output_name AS child_names_list
+            FROM
+              db_1.schema_1.[Animal] AS [Animal_1]
+              JOIN(
```

Comment:
I think this `JOIN` is a clause at the same level as the top-level `SELECT` so it should be at the same indentation level as the `SELECT`. That probably means that the rest of the query is over-indented by one stop.

Also, please use 4-space indentation here instead of 2-space, otherwise some editor programs might get confused and think that the file is 2-space indented.